### PR TITLE
Add post about my experience at the Complete Guide to Rails Performance Workshop

### PR DESCRIPTION
**Pull request by etagwerker**
Hey @rdormer @lubc, 

I was wondering if you could review my post? 

Please let me know. 

Thanks!

### CODE REVIEW

File: _posts/2018-12-20-notes-from-the-complete-guide-to-rails-performance-workshop.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+After the part about prioritizing the performance optimization work, there was
+a quick intro to the ideas of benchmarking and profiling.
+
+One of the best things about this workshop was that it wasn't just theory, for
+every topic there was a practical part to apply what you learned. For this
+section, we used `wrk` to benchmark our applications.
+
+You can quickly install `wrk` with Homebrew: `brew install wrk`
+
+Then you can quickly run a benchmark for one of your pages:
```
Comment (rdormer) [inline _posts/2018-12-20-notes-from-the-complete-guide-to-rails-performance-workshop.md]: Duplicate use of 'quickly' in two consecutive sentences

File: _posts/2018-12-20-notes-from-the-complete-guide-to-rails-performance-workshop.md [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+
+<img src="/blog/assets/images/railsperf/03-rack-mini-profiler.png" alt="Time spent in different code paths" >
+
+You can start clicking around to find more information about the execution of
+the request. If you add `?pp=profile-memory` to any request, it will show you
+the allocations grouped by gem. This feature depends on the `memory_profiler` gem.
+
+<img src="/blog/assets/images/railsperf/04-rack-mini-profiler-memory.png" alt="Memory stats for a request with Rack Mini Profiler" >
+
+If you want to see a CPU flame graph (What is a flame graph?), you can add
```
Comment (rdormer) [inline _posts/2018-12-20-notes-from-the-complete-guide-to-rails-performance-workshop.md]: Is there supposed to be a link or explanation of a flame graph here?

File: _posts/2018-12-20-notes-from-the-complete-guide-to-rails-performance-workshop.md [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+
+<img src="/blog/assets/images/railsperf/05-flamegraph.png" alt="Flamegraph provided by Rack Mini Profiler" >
+
+You can poke around to find the code paths that are most processor-intensive.
+From my experience, it can get complicated to find your own application code
+within that huge graph. You will need to zoom in to find some poorly performant
+application code.
+
+If you have a hard time interpreting the data in the flame graph, try reading
+Sam Saffron's approach to dealing with information overload.
```
Comment (rdormer) [inline _posts/2018-12-20-notes-from-the-complete-guide-to-rails-performance-workshop.md]: Link!

**Comment by etagwerker** [comment]
@rdormer Cool, thanks! I just added a few missing links.

**Pull request merged**